README.md

```diff
@@ -415,25 +410,16 @@ expansion:
 ```
 
 
-We end up with a much more verbose form of our data. In the JSON-LD specification it's called
-*expanded* form, as the original object's been expanded with a `@context`. The original object's
-form, still with an `@context`, is defined by the specification as *compacted* form.
-
-The JSON-LD parser assumes we've defined the correctly named keys for a `Person` and uses
-`http://schema.org/Person` to replace each of our properties with their more detailed schema
-definition URIs. The result is an automatically mapped set of data that uses an already available
-schema. Each key of a given value now points to a left node on a schema ontology. Leaf nodes are
```

Comment:
I think "each key points to the predicate node of an RDF triplet" is not necessarily _correct_.
After all, a predicate can just be subclassed by a domain-specific RDF ontology, hence pointing to anther part of the ontology.